recipes/newrelic/infrastructure/logs/logs.yml

```diff
@@ -126,15 +124,28 @@ install:
             done
           fi
 
+          discoveredLogFiles=$(echo -n "{{.DISCOVERED_LOG_FILES | trim}}")
```

Comment:
Where does trim come from?

Reply:
Also, why trim? I've seen error in the past with the pipe causing error in the case the variable is not defined or empty.
I would prefer to remove the trim.

Reply:
> All functions by the Go’s slim-sprig lib are available.

Reference: [`go-task` docs](https://taskfile.dev/#/usage?id=gos-template-engine) 🦙

Reply:
Ho it's Go, then maybe it's fine, I think the pipe in the bash was causing error before, but probably not in go, I assume it handles undefined variable, or does it not?

Reply:
Ohh, good call.